features/parse-html-unsafe.yml

```diff
@@ -1,3 +1,8 @@
 name: Unsanitized HTML parsing methods
 description: The `Document.parseHTMLUnsafe()` static method parses HTML into a DOM tree, while the `setHTMLUnsafe()` method of `Element` and `ShadowRoot` parses and inserts HTML into an existing tree. No sanitization applies to these methods, so never call them with user-provided HTML strings.
 spec: https://html.spec.whatwg.org/multipage/dynamic-markup-insertion.html#unsafe-html-parsing-methods
+# TODO: https://github.com/web-platform-dx/web-features/issues/1971
+# Status changed: https://github.com/web-platform-dx/web-features/pull/3200
+# 2025-08-06 - low → false - In Safari, if there are custom elements in the parsed HTML, then the elements' constructors and `connectedCallback()` methods are not called.
+# References:
+# - https://bugs.webkit.org/show_bug.cgi?id=296420
```

Comment:
I decided to let this regression through instead of overriding it. Points in favor:

- It's been Baseline low for about a year, but the bug was only recently reported. It seems like the right timing for developers to discover compat issues like this.
- This is already fixed in the Safari 26 beta and it's very likely become Baseline low again next month. It's possible that, given summer holiday and the WebDX meeting break, we wouldn't come to a conclusion on reconsideration before Apple released Safari 26 anyway.